exercises/D-logical-operators/exercise2.js

```diff
@@ -5,8 +5,34 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
-
+function isNegative(num) {
```

Comment:
This is correct! but there might be a way to make it simpler, 

hint: what does `num < 0` return?

This simplification apply to all functions that return true and false